python/lsst/ip/isr/isrTask.py

```diff
@@ -789,6 +804,27 @@ def overscanCorrection(self, exposure, amp):
             collapseRej=self.config.overscanRej,
         )
 
+    def addDistortionModel(self, exposure, camera):
+        """!Update the WCS in exposure with a distortion model based on camera geometry
+
+        \param[in,out] exposure    exposure to process; must include a Detector and a WCS;
+            the WCS of the exposure is modified in place
+        \param[in] camera  camera geometry; an lsst.afw.cameraGeom.Camera
+        """
+        self.log.info("Adding a distortion model to the WCS")
+        wcs = exposure.getWcs()
+        if wcs is None:
+            raise RuntimeError("exposure has no WCS")
+        if camera is None:
+            raise RuntimeError("camera is None")
+        detector = exposure.getDetector()
+        if wcs is None:
```

Comment:
wcs --> detector
(you've already checked for the wcs)

Also, should you move the check for the camera to the list here:
https://github.com/lsst/ip_isr/pull/42/commits/03dc329f5b1aa793166bc1f908905a97deadff38#diff-3a9973baa2fe4526d09a95e405f90269R443
i.e.
```python
if self.config.doAddDistortionModel and camera is None:
    raise RuntimeError("Must supply a camera if config.doAddDistortionModel True")
```
in order to bail out earlier?

Reply:
Thanks for catching that bug. I'll add the test you suggest. I wish it didn't have to be in 3 places, but I think you are right.